packages/zen-bull-vault/test/integration-test/EmergencyWithdraw/EmergencyRepayEulerDebtTest.t.sol

```diff
@@ -73,18 +73,18 @@ contract EmergencyRepayEulerDebtTest is Test {
 
     function testEmergencyRepayEulerDebt() public {
         uint256 ethLimitPrice =
-            UniOracle._getTwap(ETH_USDC_POOL, WETH, USDC, 420, false).wmul((ONE.sub(1e16)));
+            UniOracle._getTwap(ETH_USDC_POOL, WETH, USDC, 420, false).wmul((ONE.sub(5e15)));
```

Comment:
50bps still seems very high and probably sandwichable. I would think <10bps for any realistic size